### DISCUSSION TITLE
Laravel 9 Jetstream With Livewire  error

### DISCUSSION THREAD

**Discussion by alkoumi** [livewire/livewire#4637] [7. Bugs (archived)]
Laravel 9 Jetstream With Livewire =>  Livewire encountered corrupt data when trying to hydrate the [laravel.jetstream.http.livewire.update-profile-information-form] component. Ensure that the [name, id, data] of the Livewire component wasn't tampered with between requests.

when update any thing in user profile

**Comment by joshhanley** [comment]
@alkoumi I've just tested a new install of Laravel 9 and Jetstream and it all seems to work ok for me. 
Any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it?

Thanks!

**Reply by alkoumi** [reply]
I will try a fresh new Laravel9 today and give the feedback

**Reply by joshhanley** [reply]
Ok thanks!
